ProteoformSuiteInternal/Lollipop.cs

```diff
@@ -316,9 +338,9 @@ private static string GetOneGiantProtein(IEnumerable<Protein> proteins, bool met
         }
 
         //ET,ED,EE,EF COMPARISONS
-        public static double ee_max_mass_difference = 250; //TODO: implement this in ProteoformFamilies and elsewhere
-        public static double et_low_mass_difference = -100;
-        public static double et_high_mass_difference = 200;
+        public static double ee_max_mass_difference = 500; //TODO: implement this in ProteoformFamilies and elsewhere
```

Comment:
This is too big for unlabelled. Cant trouble shoot in reasonalble amount of time. Let's leave it at the lower values. Or make it conditional so that large range is used for neucode and small range is used for unlablelled.